cmd/infra/powervs/create.go

```diff
@@ -1001,12 +1001,9 @@ func (infra *Infra) setupPowerVSDhcp(options *CreateInfraOptions, session *ibmpi
 	// only one dhcp server is allowed per cloud instance
 	// if already a dhcp server existing in cloud instance use that instead of creating a new one
 	if len(dhcpServers) > 0 {
-		for _, dhcp := range dhcpServers {
-			log(infra.ID).Info("Using existing DHCP server present in cloud instance")
-			_, _ = client.Get(*dhcp.ID)
-			dhcpServer = &models.DHCPServerDetail{ID: dhcp.ID, Status: dhcp.Status, Network: dhcp.Network}
-			break
-		}
+		dhcp := dhcpServers[0]
```

Comment:
func should be setupPowerVSDhcp -> setupPowerVSDHCP by convention

Reply:
yes, thought of taking care in a separate PR as mentioned in other PR #1521

Reply:
Taken care here for setupPowerVSDHCP func. Will raise the PR for the whole code base soon once #1521 is merged.